crypto/engine/eng_cryptodev.c

```diff
@@ -213,29 +222,31 @@ static struct {
 static struct {
     int id;
     int nid;
-    int keylen;
+    int digestlen;
 } digests[] = {
+#  if 0
+        /* HMAC is not supported */
```

Comment:
hmac code does not exist beyond these `digests[]` entries and I have no clue how to write it so I'm removing these